service/task.go

```diff
@@ -875,13 +875,17 @@ func (uis *UIServer) testLog(w http.ResponseWriter, r *http.Request) {
 		}()
 	}
 
-	template := "task_log.html"
 	if raw {
-		template = "task_log_raw.html"
-		uis.renderText.Stream(w, http.StatusOK, data, "base", template)
-	} else {
-		uis.render.WriteResponse(w, http.StatusOK, data, "base", template)
+		if logReader != nil {
+			gimlet.WriteText(w, logReader)
+		} else {
+			uis.renderText.Stream(w, http.StatusOK, data, "base", "task_log_raw.html")
```

Comment:
I know it's from the old code, but is this supposed to actualy be test_log_raw.html?

Reply:
Yeah, there is no `test_log.html`, it would be identical if there were though. It should probably just be called `log.html` but this is legacy anyway so no point of changing it now.